README.md

```diff
@@ -36,6 +36,7 @@ Additional tools that use the libraries created by the reconciliations are also
 - `openshift-limitranges`: Manages OpenShift LimitRange objects.
 - `openshift-namespaces`: Manages OpenShift Namespaces.
 - `openshift-network-policies`: Manages OpenShift NetworkPolicies.
+- `openshift-prometheusrules`: Manages Prometheus Rules.
```

Comment:
As discussed on slack, lets rename it to `openshift-performance-parameters`

Reply:
Agreed!

Reply:
fixed in e1673c4